tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -17,6 +17,10 @@ readonly script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 
 source "${script_dir}/../../scripts/lib.sh"
 
+readonly packaging_root_dir="${repo_root_dir}/tools/packaging"
+patches_path=""
+readonly default_patches_dir="${packaging_root_dir}/kernel/patches"
```

Comment:
Imagine we're calling `get_config_version` in different files, and then we'd have to set this up in each one of the calls.

This makes me think that here is definitely not the place to add set this variables.